### PR TITLE
get all elements, new, del works

### PR DESCRIPTION
**Pull request by denisVorobyovExceed**

### CODE REVIEW

File: frontend/script.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+inputWhereSpent.addEventListener('change', event => {
+  valueInputWhereSpent = event.target.value;
+});
+
+inputHowMuch.addEventListener('change', event => {
+  valueInputHowMuch = event.target.value;
+});
+
+
```
Comment (usatovaDasha) [inline frontend/script.js:23]: delete empty line

File: frontend/script.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+window.onload = async function init() {
+  let response = await fetch('http://localhost:8080/getAllExpenses', {
+    method: 'GET'
+  });
+
+  responseJSON = await response.json();
+  if(responseJSON.data) {
```
Comment (usatovaDasha) [inline frontend/script.js:30]: replace it with a ternary condition

File: frontend/script.js [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    allExpenses = responseJSON.data;
+  } else allExpenses = [];
+
+  // console.log('all: ', responseJSON.data); // !!!!!!!!!!!!!!!!!
```
Comment (usatovaDasha) [inline frontend/script.js:34]: delete console.log

File: frontend/script.js [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+  const howMuch = document.createElement('span');
+  howMuch.id = `item-span-money-${index}`;
+  howMuch.className = 'items-span-money';
+  howMuch.innerHTML = `${item.howMuch} &#8381;`;
+  itemExpense.appendChild(howMuch);
+
+  const buttons = document.createElement('div');
+  buttons.className = 'items-buttons-div';
+
+    const imgEdit = document.createElement('img');
```
Comment (usatovaDasha) [inline frontend/script.js:92]: why the large indent?

File: frontend/script.js [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+        whereSpent: valueInputWhereSpent,
+        howMuch: valueInputHowMuch
+      })
+    });
+    
+    inputWhereSpent.value = '';
+    inputHowMuch.value = '';
+
+    responseJSON = await response.json();
+    if(responseJSON.data) {
```
Comment (usatovaDasha) [inline frontend/script.js:141]: code style!

File: frontend/script.js [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+      allExpenses = responseJSON.data;
```
Comment (usatovaDasha) [inline frontend/script.js:142]: replace it with a ternary condition

File: frontend/script.js [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+  // TO DO
+}
+
+deleteItem = async (index) => {
+  const response = await fetch(`http://localhost:8080/deleteExpense?id=${allExpenses[index]._id}`, {
+      method: 'DELETE',
+  });
+  
+  responseJSON = await response.json();
+  if(responseJSON.data) {
```
Comment (usatovaDasha) [inline frontend/script.js:185]: replace it with a ternary condition

File: frontend/style.css [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+  height: 24px;
+  font-family: Roboto;
+  font-size: 13pt;
+  border-radius: 5px;
+  border: 1px solid #88898C;
+  outline: none;
+}
+
+.what {
+  width: 365px;
```
Comment (usatovaDasha) [inline frontend/style.css:50]: we don't use px. you can use %, vw, vh

File: src/modules/task.controller.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+  expense.save().then(() => {
+    Expense.find().then(result => {
+      res.send({data: result});
+    });
+  });
+};
+
+module.exports.deleteExpense = (req, res, next) => {
+  Expense.findByIdAndDelete(req.query.id).then(() => {
```
Comment (usatovaDasha) [inline src/modules/task.controller.js:23]: why don't deleteOne?

File: src/modules/task.controller.js [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+    Expense.find().then(result => {
+      res.send({data: result});
+    });
+  });
+};
+
+module.exports.editExpense = (req, res, next) => {
+  Expense.findByIdAndUpdate(req.body).then(() => {
```
Comment (usatovaDasha) [inline src/modules/task.controller.js:31]: why don't updateOne?

File: src/modules/task.controller.js (hunk 9 above)
Comment (denisVorobyovExceed) [reply src/modules/task.controller.js:23]: I sent here just document's _id, not whole object. Can replace it with ...deleteOne({_id: req.query.id})...

**Pull request merged**